utils/support_utils.py

```diff
@@ -30,6 +30,10 @@ def create_submodule(device_info, create_submodule_in_git):
     copyfile(SUPPORT_MAKEFILE, path.join(device_info.support_dir(), "Makefile"))
     master_dir = device_info.support_master_dir()
     if create_submodule_in_git:
+        if path.isdir(path.join(master_dir, ".git")):
+            logging.error("A git repository (not submodule) already exists at {0}."
```

Comment:
Is this right? Should it be    path.join(master_dir, device_info.support_app_name(), ".git")   ?

Reply:
Sorry, think it is fine